aiogram/utils/text_decorations.py

```diff
@@ -57,14 +57,14 @@ def unparse(self, text: str, entities: Optional[List[MessageEntity]] = None) ->
         """
         result = "".join(
             self._unparse_entities(
-                text, sorted(entities, key=lambda item: item.offset) if entities else []
+                text.encode('utf-16-le'), sorted(entities, key=lambda item: item.offset) if entities else []
             )
         )
         return result
 
     def _unparse_entities(
         self,
-        text: str,
+        text: bytes,
```

Comment:
I think text should be passed to this function without any changes and then encoding should be placed inside this method